app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -88,7 +88,8 @@ def post_front_image
       end
 
       def test_credentials?
-        FeatureManagement.allow_doc_auth_test_credentials? && image.content_type == 'text/x-yaml'
+        FeatureManagement.allow_doc_auth_test_credentials? &&
+          ['text/x-yaml', 'text/plain'].index(image.content_type)
```

Comment:
You may have better luck with `#includes?`. `0` is falsy.

Reply:
True and 0 is still good

Reply:
true && 0 => thumbs up.  true && nil => no dice

Reply:
Ah interesting, 0 is truthy. We could still change it though to be a little more semantic.

Reply:
done